dev-cpp/asio/files/asio-1.30.1-pkgconfig.patch

```diff
@@ -0,0 +1,13 @@
+Install .pc file to /usr/share instead of /usr/lib because there are no .so files
+
+--- a/Makefile.am
++++ b/Makefile.am
+@@ -1,7 +1,7 @@
+ AUTOMAKE_OPTIONS = foreign dist-bzip2 dist-zip
+ 
+ pkgconfig_DATA = asio.pc
+-pkgconfigdir = $(libdir)/pkgconfig
++pkgconfigdir = $(datadir)/pkgconfig
```

Comment:
delete the entire line, and call `PKG_NOARCH_INSTALLDIR` in `configure.ac`, and then use `noarch_pkgconfig_DATA` above

Reply:
Cool